Updating guidelines unofficial source (#6428)
simple-icons/simple-icons: CONTRIBUTING.md

    - For websites, the [Alexa rank](https://www.alexa.com/siteinfo) should be less than 500k.
        - Note that for brands that have already been added the threshold for continued inclusion rises to 750k.
    - For GitHub projects, the amount of "stars" should be above 5k.

>     - For a well known or widely accepted de facto standard logo (where official sources don't exist), popularity will be assesed on a case-by-case basis by maintainers. For example, Javascript falls in this category.

Reviewer: > For a well known or widely accepted de facto standard logo (where official sources don't exist)

I think we should pick one from "well known," "widely accepted," "de facto," and "standard," instead of all four, to make the language simpler.

Author: make the choice @sachinraja. I don't really prefer any over the others

Reviewer: "widely accepted" makes the most sense to me.

Reviewer 2: > "widely accepted" makes the most sense to me

To me too.

Author: widely accepted it is, then.

>     - For a well known or widely accepted de facto standard logo (where official sources don't exist), popularity will be assesed on a case-by-case basis by maintainers. For example, Javascript falls in this category.

Reviewer: ```suggestion
    - For a well known or widely accepted de facto standard logo where official sources don't exist, popularity will be assesed on a case-by-case basis by maintainers. For example, Javascript falls in this category.
```
This information is necessary, so I don't think the parentheses should be here.

>     - For anything else, popularity will be judged on a case-by-case basis.

Reviewer 3: This feels to me like we're repeating ourselves, both bullets say "popularity will be judged on a case-by-case basis". I don't think the separate bullet is necessary. Especially as the extra bullet is (still) not primarily about the popularity but rather about the logo.

Author: it is indeed the same info. I did it on purpose to make the message extra clear.

Reviewer 3: I personally disagree that it makes things extra clear, but I understand it's highly subjective. The reason I disagree is 1) now there is more text to read, 2) the added sentence isn't easy to read (at least compared to the existing sentence), and 3) again, this makes the list inconsistent (the focus changes from popularity to where the logo comes from).

In fact, I would argue it may cause confusion. The repetitions reads, to me, as if either 1) not much care was put into creating a coherent document making me less inclined to read it fully, or 2) one of the bullets is incorrect and does not apply.

Author: I disagree.
point 3) the list is not inconsistent. we are still talking about popularity. The previous bullets were also talking about the metrics of popularity and their sources. I think we are staying on topic. At least as much as the last bullet is.

point 2) I welcome rewrites. I've already committed yours and @sachinraja suggestions for more concise text.

point 1 and second paragraph:  Not at all. You are assuming people are going to read the full document top to bottom. While this is going to be true of some people, most people will just refer to the information they need, be it accepted sources and guidelines, or popularity metrics.
In any case, reiterating a point is not necessarily bad writing. Making the point clear in the places an user may jump into is useful. When the user don't have to scan the whole text, it's more likely that they will contribute and not be overwhelmed.

An alternative way of doing this would be to link one text to the other. Perhaps removing the bullet in the popularity list and adding a link at the end of the last one:
`- For anything else, popularity will be judged on a case-by-case basis. (for example, an unofficial de facto standard logo)` where the text in parenthesis would point to the sources text:`It may be the case that no official source exist...`

Reviewer: I don't think this should be mentioned in the popularity section as the last point already covers the case. The statement at the end (It may be the case...) can explain the policy for unofficial icons.

Reviewer 3: Again, highly subjective. I don't mind it being there but from my point of view I would just recommend not putting it there. I think @sachinraja put it best in their comment.

<details>
<summary>Just a few responses to @jorgeamadosoria</summary>

> we are still talking about popularity

Yes, but the **focus** of the sentence is on the icon being unofficial.

> I welcome rewrites

I don't think any amount of rewrite can fix my problem with this bullet.

> You are assuming people are going to read the full document top to bottom

That was not my assumption - though I must admit my comment wasn't well written. I should've said something along the lines of: "less inclined to follow the guidelines".
</details>

Author: Removed bullet point.

1. Wikimedia (which should provide a source)
1. GitHub repositories

> It may be the case that no official source exist, but an unofficial icon has gained widespread acceptance and popularity. In such cases the unofficial icon can be included, but the details will be judged on a case-by-base basis. The JavaScript icon is an example of this

Reviewer 2: Just a typo:

```suggestion
It may be the case that no official source exist, but an unofficial icon has gained widespread acceptance and popularity. In such cases the unofficial icon can be included, but the details will be judged on a case-by-base basis. The JavaScript icon is an example of this.
```

> It may be the case that no official source exist, but an unofficial icon has gained widespread acceptance and popularity. In such cases the unofficial icon can be included, but the details will be judged on a case-by-base basis. The JavaScript icon is an example of this.

Reviewer: ```suggestion
It may be the case that no official source exists, but an unofficial icon has gained widespread acceptance and popularity. In such cases the unofficial icon can be included, but the details will be judged on a case-by-case basis. The JavaScript icon is an example of this.
```

Just two grammar fixes

Author: good catch! Committed the changes already.

1. Wikimedia (which should provide a source)
1. GitHub repositories

> It may be the case that official sources simply do not exist, but an unofficial icon has gained widespread acceptance and popularity. These icons can also be included, after being reviewed and accepted by maintainers on a case-by-base basis. Javascript is the prime example of this situation.

Reviewer 3: A couple suggestions, some for brevity and some for clarity:

```suggestion
It may be the case that no official source exist, but an unofficial icon has gained widespread acceptance and popularity. In such cases the unofficial icon can be included, but the details will be judged on a case-by-base basis. The JavaScript icon is an example of this
```

Reviewer: "exist" would also be changed to "exists"

> It may be the case that official sources simply do not exist, but an unofficial icon has gained widespread acceptance and popularity. These icons can also be included, after being reviewed and accepted by maintainers on a case-by-base basis. Javascript is the prime example of this situation. 

Reviewer 2: The linting is failing due to this trailing whitespace, could you remove it?

```suggestion
It may be the case that official sources simply do not exist, but an unofficial icon has gained widespread acceptance and popularity. These icons can also be included, after being reviewed and accepted by maintainers on a case-by-base basis. Javascript is the prime example of this situation.
```

Author: Committed suggestion

> Notice that an unofficial icon won't replace or supersede an official one, even if it more popular. An unofficial icon will only be accepted if official options do not exist.

Reviewer 3: A couple suggestions for brevity:

```suggestion
Notice that an unofficial source will never supersede an official one, even if it is more popular. An unofficial icon will only be accepted if no official option exists.
```

Reviewer: "exist" would also be changed to "exists"
